src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.File.NotifyFilter.cs

```diff
@@ -251,6 +251,26 @@ public void FileSystemWatcher_File_NotifyFilter_LastWriteAndFileName()
             }
         }
 
+        /// <summary>
+        /// Tests that a filename with just a space does not raise a watcher changed event.
+        /// </summary>
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void FileSystemWatcher_File_NotifyFilter_FileWithSpaces()
+        {
+            using (var testDirectory = new TempDirectory(GetTestFilePath()))
+            using (var file = new TempFile(Path.Combine(testDirectory.Path, " ")))
```

Comment:
We should actually raise the event. A filename of " " is valid on Windows as well as Unix, it's just hard to create. I'm sort of confused how the `TempFile` is getting created here as it shouldn't without using `\\?\`.

Reply:
Correct me if I'm wrong, but even though a file w/ spaces is valid in Windows, IsEffectivelyEmpty will prevent the event from being raised.

Are you suggesting a change in fsw?

Reply:
> IsEffectivelyEmpty will prevent the event from being raised.

Where do you see that happening? I just did a quick look and I'm not seeing this sort of filter in the FSW code.